udata/core/dataset/rdf.py

```diff
@@ -133,7 +133,6 @@ def resource_to_rdf(resource, dataset=None, graph=None):
     r.add(DCT.description, Literal(resource.description))
     r.add(DCAT.downloadURL, URIRef(resource.url))
     r.add(DCAT.accessURL, URIRef(permalink))
-    r.add(DCT.issued, Literal(resource.published))
```

Comment:
Maybe keep this line but using `created_at`. Maybe treat it in https://github.com/opendatateam/udata/pull/2810, since we could want to use harvest info as well on the different dates exposed in RDF.